.changeset/little-colts-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphcms/rich-text-react-renderer': patch
+---
+
+Update test
```

Comment:
Shall we delete this file? I think if it's only a test update, there's no need to release a new version on npm.

Reply:
Sure!